conda_build/index.py

```diff
@@ -937,7 +939,7 @@ def index_subdir(self, subdir, index_file=None, verbose=False, progress=False):
             new_repodata_packages = {k: v for k, v in old_repodata.get('packages', {}).items() if k in unchanged_set}
             new_repodata_conda_packages = {k: v for k, v in old_repodata.get('packages.conda', {}).items() if k in unchanged_set}
 
-            for k in unchanged_set:
+            for k in sorted(unchanged_set):
                 if not (k in new_repodata_packages or k in new_repodata_conda_packages):
```

Comment:
This is still gross because both `new_repodata_packages` and `new_repodata_conda_packages` are dicts.  These should both be sets as well.

Reply:
nvm